app/javascript/components/analytics/widgets/MonitoreeFlow.js

```diff
@@ -16,14 +16,45 @@ class MonitoreeFlow extends React.Component {
         let thisTimeFrameData = props.stats.monitoree_snapshots.find(
           monitoree_snapshot => monitoree_snapshot.status === workflow && monitoree_snapshot.time_frame === time_frame
         );
+        let exposure = workflow === 'Exposure';
+        let inTotal =
+          thisTimeFrameData?.new_enrollments +
+          thisTimeFrameData?.transferred_in +
+          (exposure ? thisTimeFrameData?.isolation_to_exposure : thisTimeFrameData?.exposure_to_isolation);
+        let outTotal =
+          thisTimeFrameData?.closed +
+          thisTimeFrameData?.transferred_out +
+          (exposure ? thisTimeFrameData?.exposure_to_isolation : thisTimeFrameData?.isolation_to_exposure);
         return {
           time_frame,
-          new_enrollments: thisTimeFrameData?.new_enrollments || 0,
-          transferred_in: thisTimeFrameData?.transferred_in || 0,
-          closed: thisTimeFrameData?.closed || 0,
-          transferred_out: thisTimeFrameData?.transferred_out || 0,
-          exposure_to_isolation: thisTimeFrameData?.exposure_to_isolation || 0,
-          isolation_to_exposure: thisTimeFrameData?.isolation_to_exposure || 0,
+          new_enrollments: {
```

Comment:
We have an [analytics utility file](https://github.com/SaraAlert/SaraAlert/blob/master/app/javascript/utils/Analytics.js) for random re-used bits of code. The formatting of all these statements might make for a great utility function in there! Plus it would make this much more readable

Reply:
Point taken, but in this case I think it's better to keep it where it is. Reasoning is that the logic in calculating these percentages is very specific to `MonitoreeSnapshot` and the way that model is presented in monitoree flow table, i.e. not very reusable. And in cases like that I think it's better to have the logic closer to the place where it's used.

Reply:
Yeah I was actually thinking about this this morning. I do agree that this probably doesnt belong in the analytics utility helper. What are your thoughts about a formatting function within this component then?

Reply:
I'm not sure what you mean by a formatting function. Do you mean a function for the `<td>`s below that contain the # and % ?

Reply:
I actually agree with @DPC15038 on this one, I'd like to see a formatter for the percent because you use that in a ton of places.  Makes sense to go in the analytics utility helper.  Something like:

```function formatPercent(count, total) {
  return count ? ((count / total) * 100).toFixed(1).toString() + '%' : 'None'
}```

Reply:
Ah ok yeah, I agree with that. Was going to say I don't want to put it in a a separate utils file if this is the only place I'm using it, but It's actually the same logic that goes into the percentages in the [preferred reporting methods table](
https://github.com/SaraAlert/SaraAlert/blob/1.29.0/app/javascript/components/analytics/widgets/PreferredReportingMethod.js#L75), so I'll do that.

Reply:
Done here https://github.com/SaraAlert/SaraAlert/pull/864/commits/b8b4afbe350e21ef92819546fd60883cd7a45214